wercker.yml

```diff
@@ -0,0 +1,10 @@
+box: centos:6
+build:
+  steps:
+    - script:
+        name: install rmagick
+        code: |
+          sudo yum install -y ruby ruby-devel rubygems ImageMagick-devel gcc
```

Comment:
I realize now that installing the latest version makes no sense in terms of CI which this is. We really have to run the tests just like on travis. `rake-compiler` runs the same compilation process anyway. Could you commit it to the `wercker` branch?